test/models/user_test.rb

```diff
@@ -0,0 +1,56 @@
+require 'test_helper'
+
+class UserTest < ActiveSupport::TestCase
+  def setup
+    @user = User.new(name: 'Example User', email:'user@example.com')
+  end
+
+  test 'should be valid' do
+    assert @user.valid?
+  end
+
+  test 'name should be present' do
+    @user.name = ''
+    assert_not @user.valid?
+  end
+
+  test 'email should be present' do
+    @user.email = ''
+    assert_not @user.valid?
+  end
+
+  test 'name should not be too long' do
+    @user.name = 'a' * 51
+    assert_not @user.valid?
+  end
+
+  test 'email should not be too long' do
+    @user.email = 'a' * 244 + '@example.com'
+    assert_not @user.valid?
+  end
+
+  test 'email validation should accept valid addresses' do
+    valid_addresses = %w[user@example.com USER@foo.COM A_US-ER@foo.bar.org
+                first.last@foo.jp alice+bob@baz.cn]
+    valid_addresses.each do |valid_address|
+      @user.email = valid_address
+      assert @user.valid?, "#{valid_address.inspect} should be valid"
+    end
+  end
+
+  test 'email validation should reject invalid addresses' do
+    invalid_addresses = %w[user@example,com user_at_foo.org user.name@example.
+                foo@bar_baz.com foo@bar+baz.com]
```

Comment:
__[rubocop]__
* Align the elements of an array literal if they span more than one line. [:ref](https://github.com/bbatsov/ruby-style-guide#align-multiline-arrays)